src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -61,42 +65,96 @@ public async Task<bool> PushMetadataAsync()
                 }
                 else
                 {
-                    List<BuildData> buildsManifestMetadata = GetBuildManifestsMetadata(ManifestsPath);
+                    List<BuildData> buildsManifestMetadata = GetBuildManifestsMetadata(ManifestsPath, cancellationToken);
 
                     BuildData finalBuild = MergeBuildManifests(buildsManifestMetadata);
 
                     IMaestroApi client = ApiFactory.GetAuthenticated(MaestroApiEndpoint, BuildAssetRegistryToken);
-                    Client.Models.Build recordedBuild = await client.Builds.CreateAsync(finalBuild, s_cancellationToken);
+
+                    var deps = await GetBuildDependenciesAsync(client, cancellationToken);
+                    Log.LogMessage(MessageImportance.High, "Calculated Dependencies:");
```

Comment:
>High [](start = 53, length = 4)

High is pretty high. :-) But maybe it's fine, since these run in the release these days, right?

Reply:
These currently run in the build, as part of the final "Publish to Build Asset Registry" step. It doesn't run on dev machines.